fix(stdlib): Fix parsing nginx error log delaying request (#1285)
vectordotdev/vrl: changelog.d/1285.fix.md

> Fix `parse_nginx_log` function when a format is set to error and parsing `delaying requests` message.

Reviewer: ```suggestion
The `parse_nginx_log` function can now parse `delaying requests` error messages.
```
